doc/user_doc/vic_installation/plugin_vc_web.md

```diff
@@ -1,22 +1,24 @@
-# Install the vSphere Integrated Containers Engine Plug-In on vCenter Server For Windows by Using a Web Server #
+# Install the Flex Plug-In on vCenter Server for Windows by Using a Web Server #
 
-If your vCenter Server instance runs on Windows, you can use a Web server to host the vSphere Web Client plug-in for vSphere Integrated Containers Engine.
+If your vCenter Server instance runs on Windows, you can use a Web server to host the Flex-based vSphere Web Client plug-in for vSphere Integrated Containers Engine.
 
 **Prerequisites**
 
 - You deployed at least one virtual container host (VCH) to a vCenter Server instance that runs on Windows.
 - You are running a Web server that your vCenter Server instance can access.
 - You must use a Windows system to run the script to install the plug-in on a vCenter Server that runs on Windows. If you used a Linux or Mac OS system to deploy the VCH, download and unpack the vSphere Integrated Containers Engine package on a Windows system. For example, download the package to the system on which vCenter Server is running.
-- If you deployed the VCH to a vCenter Server 6.5 instance, use the Flash-based vSphere Web Client to view the vSphere Web Client plug-in for vSphere Integrated Containers Engine. vSphere Integrated Containers Engine does not currently provide a plug-in for the new HTML5 vSphere Client.
 
 **Procedure**
 
 1. On the Windows system on which you have downloaded and unpacked vSphere Integrated Containers Engine, navigate to the folder that contains the `vic-machine` utility and open the `ui` folder.
 2. Upload the plug-in bundle to your Web server.
-    <pre><i>unpack_dir</i>\vic\ui\vsphere-client-serenity\com.vmware.vicui.Vicui-<i>version</i>.zip</pre>
+    <pre><i>unpack_dir</i>\vic\ui\vsphere-client-serenity\com.vmware.vic-<i>version</i>.zip</pre>
 3. On the `vic-machine` system, open the <code><i>vic_unpack_dir</i>\vic\ui\vCenterForWindows\configs</code> file in a text editor.
 4. Enter the IPv4 address or FQDN of the vCenter Server instance on which to install the plug-in.<pre>SET target_vcenter_ip=<i>vcenter_server_address</i></pre>
 5. Enter the path to the folder on your Web server that contains the <code>com.vmware.vicui.Vicui-<i>version</i>.zip</code> file.<pre>SET vic_ui_host_url="<i>vicui_zip_location</i>"</pre>
```

Comment:
<code>com.vmware.vicui.Vicui-<i>version</i>.zip</code> should be <code>com.vmware.vic.ui-<i>version</i>.zip</code>